### PR TITLE
fix for cust sys packages path

### PR DESCRIPTION
**Pull request by hallvictoria**
## Description
Changes in worker and constants to fix "Customer packages not in sys path." warning log being hit.

Fixes #

---

### PR information

- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### Quality of Code and Contribution Guidelines
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-functions-python-worker/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: azure_functions_worker/constants.py [hunk 1]
```diff
@@ -56,4 +56,4 @@
 # Paths
 CUSTOMER_PACKAGES_PATH = os.path.join(pathlib.Path.home(),
                                       pathlib.Path(
-                                          "site/wwwroot/.python_packages"))
+    "site/wwwroot/.python_packages/lib/site-packages"))
```
Comment (vrdmr) [inline azure_functions_worker/constants.py:59]: The indentation - is flake working?

File: python/prodV4/worker.py [hunk 2]
```diff
@@ -1,10 +1,9 @@
 import os
+import pathlib
 import sys
 
-from pathlib import Path
```
Comment (vrdmr) [inline python/prodV4/worker.py:4]: Any specific reason to remove it?

File: azure_functions_worker/constants.py (hunk 1 above)
Comment (vrdmr) [reply azure_functions_worker/constants.py:59]: Also - adding /lib/site-packages - would it work? we should also explore if we need another check if site/wwwroot/.python_packages/ exists

And why not add /home here as well? to simplify it

Comment (hallvictoria) [reply azure_functions_worker/constants.py]: For python 3.7+ customer packages should be at /home/site/wwwroot/.python_packages/lib/site-packages, so this should fix the issue

File: python/prodV4/worker.py (hunk 2 above)
Comment (hallvictoria) [reply python/prodV4/worker.py:4]: Making it more consistent with the other imports in that file, and "import pathlib" and "pathlib.Path" was how the module was used in constants.py before

**Review by gavin-aguiar — APPROVED** [review]

**Pull request merged**